public/assets/materialize/pushpin-21271bd46669449b24c183353ec4f9432a1fb8d145642b7c977942de990ca93f.js

```diff
@@ -0,0 +1 @@
+!function(t){t(document).ready(function(){t.fn.pushpin=function(s){var o={top:0,bottom:1/0,offset:0};return s=t.extend(o,s),$index=0,this.each(function(){function o(t){t.removeClass("pin-top"),t.removeClass("pinned"),t.removeClass("pin-bottom")}function i(i,n){i.each(function(){s.top<=n&&s.bottom>=n&&!t(this).hasClass("pinned")&&(o(t(this)),t(this).css("top",s.offset),t(this).addClass("pinned")),n<s.top&&!t(this).hasClass("pin-top")&&(o(t(this)),t(this).css("top",0),t(this).addClass("pin-top")),n>s.bottom&&!t(this).hasClass("pin-bottom")&&(o(t(this)),t(this).addClass("pin-bottom"),t(this).css("top",s.bottom-a))})}var n=Materialize.guid(),e=t(this),a=t(this).offset().top;i(e,t(window).scrollTop()),t(window).on("scroll."+n,function(){var o=t(window).scrollTop()+s.offset;i(e,o)})})}})}(jQuery);
```

Comment:
Line is too long.<br>Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.<br>'i' is already defined.<br>Wrap an immediate function invocation in parens to assist the reader in understanding that the expression is the result of a function, and not the function itself.<br>'$index' is not defined.<br>'Materialize' is not defined.